repos/exawind/packages/trilinos/package.py

```diff
@@ -9,15 +9,15 @@
 from spack.pkg.builtin.trilinos import Trilinos as bTrilinos
 import os
 import importlib
-from spack.pkg.exawind.cmake_extension import *
+from spack.pkg.exawind.ctest_package import *
 find_machine = importlib.import_module("find-exawind-manager")
 
-class Trilinos(bTrilinos, CmakeExtension):
+class Trilinos(bTrilinos, CtestPackage):
     # Our custom release versions should be the latest release tag found on
     # the trilinos github page appended with the date of the commit.
     # this preserves the Trilinos versioning scheme and will allow for valid
     # version comparisons in spack's internals.
-    version("14.1.0.2023.02.28", commit="8b3e2e1db4c7e07db13225c73057230c4814706f")
+    version("13.4.1.2023.02.28", commit="8b3e2e1db4c7e07db13225c73057230c4814706f")
```

Comment:
I think this requires changes in machine configs too.

Reply:
Yeah you're right.